component/myblogs/MyBlogs.jsx

```diff
@@ -32,7 +32,7 @@ export default function MyBlogs({ posts }) {
                     </a>
                   </Link>
                   <div className={`text-xs text-gray-400`}>
-                    {moment(item.updatedAt).format("LL")}
+                    {moment(item.created_at).format("LL")}
```

Comment:
the date shown should be the `published_at`.

Reply:
published_at is not available for all blogs as some are in drafted mode

Reply:
oh. my bad. THis was my blogs :+1: